gradleup.groovy

```diff
@@ -1,25 +1,29 @@
 import groovy.json.*
 
 def api = new JsonSlurper().parse(System.in)
+def wrapperConfig = new File("gradle/wrapper/gradle-wrapper.properties")
 
 def version = api.name
-def lines = new File("gradle/wrapper/gradle-wrapper.properties").readLines()
+def lines = wrapperConfig.readLines()
+def distributionType = "bin"
 
 for (int i = 0; i < lines.size(); i++) {
     if (lines[i].startsWith('distributionUrl')) {
-        lines[i] = "distributionUrl=https\\://services.gradle.org/distributions/gradle-${version}-bin.zip".toString()
+        if (lines[i].contains("-all")) {
+            distributionType = "all"
+        }
+        lines[i] = "distributionUrl=https\\://services.gradle.org/distributions/gradle-${version}-${distributionType}.zip".toString()
     }
 }
 
-new File("gradle/wrapper/gradle-wrapper.properties").text =
-        lines.join('\n') + '\n'
+wrapperConfig.text = lines.join('\n') + '\n'
 
 new File('/tmp/commit.txt').text =
-        "Update gradle wrapper to ${version}\n\n${api.body}"
+        "Gradle wrapper updated to ${version}\n\n${api.body}"
```

Comment:
I can understand why many would find imperative too strong here. But it's there for consistency. Git merge and revert events use imperative. I think it should go back to the imperative for this and PR Json being built below.

Reply:
Understand, rollback to `Update gradle wrapper`?

Reply:
Yep. Sounds good.

Reply:
Imperative is the current style. See https://github.com/joelparkerhenderson/git_commit_message#summary-keywords and https://chris.beams.io/posts/git-commit/#imperative